docs/project/overview.md

```diff
@@ -4,66 +4,78 @@
+**Educado** is a web and mobile application developed as part of the **Egalitarian Erasmus Initiative**, an effort within the **Egalitarian SDG Challenge** and supported by the **Erasmus+ Programme**. The project involves collaboration among students from:
 
-The project's aim is to reduce downtime, increase operational efficiency, and minimize the need for manual intervention in server management, ultimately improving the overall stability of IT systems.
+- **University of Brasília (Brazil)**  
+- **Aalborg University (Denmark)**  
+- **University of Minho (Portugal)**  
+- **Saxion University (Netherlands)**  
+
+Educado is designed to **empower waste pickers** by providing educational resources that improve their financial literacy, health, machine operation skills, and hygiene practices. Teachers can upload educational content through a web app, while waste pickers can access the classes through a mobile app. By equipping waste pickers with accessible learning tools, the project contributes to **Sustainable Development Goal (SDG) 4: Quality Education**.
 
 ---
 
 ### Objectives and Scope
 
 #### Objectives:
 
-* Develop a scalable and efficient real-time monitoring system capable of tracking server health across a wide range of metrics (e.g., CPU, memory, disk usage, network traffic).
-* Implement machine learning algorithms to analyze historical and real-time server data, detecting anomalies and predicting potential hardware failures.
-* Design an alerting system that notifies administrators of detected issues through various channels, including email, SMS, and integration with messaging platforms like Slack.
-* Create an intuitive dashboard to provide administrators with real-time insights into server status, trends, and health indicators, allowing for informed decision-making.
+* Develop a **web platform for teachers** to upload educational videos and materials.
+* Develop a **mobile application** that delivers **educational content** tailored for waste pickers.
+* Improve **financial literacy, health awareness, and professional skills** within the waste-picking community.
+* Provide **easy-to-understand learning materials** through multimedia content.
+* Promote **digital inclusion** by ensuring accessibility and usability of the application.
 
 #### Scope:
 
 **Included:**
 
-* Real-time data collection from servers running various operating systems (Linux, Windows, etc.).
-* Integration with existing IT infrastructure tools such as Nagios, Zabbix, and Prometheus.
-* Machine learning models to detect performance anomalies and predict potential failures based on historical data.
-* Alerting system with customizable thresholds and multi-channel notifications.
-* Visual dashboard for displaying server health and performance metrics.
+- **Web platform for teachers**: enables educators to upload videos, documents, and educational materials.
```

Comment:
not only teachers. Usar algum termo que defina esse usuário. Como por exemplo "Content Creator"